src/Domain/Structure/Neighbors.cpp

```diff
@@ -67,16 +68,37 @@ bool operator!=(const Neighbors<VolumeDim, IdType>& lhs,
 
 template <size_t VolumeDim, typename IdType>
 void Neighbors<VolumeDim, IdType>::pup(PUP::er& p) {
-  p | ids_;
-  p | orientation_;
+  if constexpr (std::is_same_v<IdType, size_t>) {
+    size_t version = 1;
+    p | version;
```

Comment:
Add a comment that version 0 is deserializing the old `BlockNeighbor` class. That's what's happening here, right?